### DISCUSSION TITLE
MHI2_JP_AU37x_P0902 update and EU conversion

### DISCUSSION THREAD

**Discussion by augis1999** [Mr-MIBonk/M.I.B._More-Incredible-Bash#228] [General]
Hello, not sure if it is the place to ask.

I own Audi A3 2018 8V with MMI version: MHI2_JP_AU37x_P0902. I did not find my version in MIB compatible patches, is anyone aware if I would be able to patch my MMI with MIB and convert it to EU, so I would be able to update to the latest EU maps?

Thank You!
Regards!

**Comment by harman-f** [comment]
A backup of that FW would be helpful--> zip --> filehorst.de

EU conversion is possible afterwards.
